openc3/lib/openc3/io/json_api.rb

```diff
@@ -65,7 +65,7 @@ def _generate_auth
 
     def _request(*method_params, **kw_params)
       kw_params[:scope] = $openc3_scope unless kw_params[:scope]
-      kw_params[:json] => true unless kw_params[:json]
+      kw_params[:json] = true unless kw_params[:json]
```

Comment:
This previously looks like a syntax error. Not even sure what this does. Did you look for other examples?

Reply:
It was a bug for sure.  Setting the json flag adds the appropriate content-type header.